data/src/main/java/com/architect/coders/mu8/data/events/EventsEntity.kt

```diff
@@ -0,0 +1,23 @@
+package com.architect.coders.mu8.data.events
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.architect.codes.mu8.events.Event
+import com.architect.codes.mu8.utils.EMPTY_LONG
+import com.architect.codes.mu8.utils.EMPTY_STRING
+
```

Comment:
check if do you need select a specific event.

Reply:
Done